components/atoms/NegativeButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useColorScheme } from 'react-native'
+import { Button, type ButtonProps } from 'tamagui'
+
+import getColors from '~/constants/Colors'
+import { RADIUS_BUTTON } from '~/constants/Constants'
+
+type Props = {
+  title: string
+  icon?: JSX.Element
```

Comment:
The icon property is already available in [ButtonProps](https://tamagui.dev/ui/button#api-reference), there is no need to add an icon property here. Delete it

```suggestion

  type Props = {
  title: string
  } & ButtonProps

```